package.json

```diff
@@ -105,7 +106,47 @@
       "prettier --write"
     ]
   },
+  "jest": {
+    "roots": [
+      "<rootDir>/src"
+    ],
+    "globals": {
+      "ts-jest": {
+        "babelConfig": true,
+        "tsConfig": "<rootDir>/tsconfig.json"
+      }
+    },
+    "moduleFileExtensions": [
+      "ts",
+      "tsx",
+      "js"
+    ],
+    "setupFilesAfterEnv": [
+      "<rootDir>/setupTests.tsx"
+    ],
+    "testPathIgnorePatterns": [
+      "<rootDir>/node_modules/",
+      "<rootDir>/src/database/"
+    ],
+    "transform": {
+      "^.+\\.tsx?$": "ts-jest"
+    },
+    "moduleNameMapper": {
+      ".+\\.(svg|css)$": "<rootDir>/src/__mocks__/svgMock.ts"
+    }
+  },
```

Comment:
This config is fairly extensive. Can you document where you got some of these snippets from? This should help developers modify it in the future (hint: will help me when adding testing to Gatsby)!

Reply:
@daschi oops just saw that you merged it. You can just comment it here!

Reply:
Oops--I'm sorry @szhu I merged before seeing this comment! I mostly looked at [jest](https://jestjs.io/docs/en/getting-started#using-typescript) and [ts-jest](https://kulshekhar.github.io/ts-jest/user/config/) documentation.